email.html

```diff
@@ -37,71 +31,23 @@
 
 <HR NOSHADE SIZE=1>
 
-<!-- <CENTER> -->
-
-
-<!--
-<B>Please note that I will not be able to respond to email
- from June 28 through July 5.
-- Norm</B><P>
--->
-
-
-You can email Norman Megill at
-"<FONT COLOR=BLUE>nm at alum period mit period edu</FONT>" with
-"at" and "period" replaced by the corresponding characters.
-
-
-<P> Your first email should include "Metamath" in the subject to add you
-to my spam filter's whitelist.
-
-
-<!--
-<P>You can also contact me
- via <A HREF="http://xri.net/=nm">=nm</A> (which is my permanent
-<A HREF="http://en.wikipedia.org/wiki/I-name">i-name</A>).
--->
-<!-- <A HREF="http://xdi.org/">i-name</A>). -->
-<!--
-<B>Include your email address inside your message, because
-otherwise I will have no way to contact you.</B>
--->
-
-
-<P> When possible, I prefer plain text email (not <A
-HREF="http://www.efn.no/html-bad.html">HTML formatted</A>).
-
-<P>
-<HR NOSHADE SIZE=1>
-
-<B>For gmail.com users</B>
-
-<P>As of September, 2018, Google seems to have put me on some kind of
-blacklist and has been bouncing many of my emails to gmail.com as spam.
-The following procedure to add me (nm at alum dot mit dot edu) to
-gmail's white list seems to have solved the problem in one case:
-
-<P><A HREF="https://www.labnol.org/internet/gmail-whitelist/19754/"
->How to Setup a Whitelist in Gmail</A>
-
-<P>I would recommend doing that if you email me from a gmail
-address (and want a response).
-
+<p>The two best ways to contact the metamath project are:</p>
 
- <P> <HR NOSHADE SIZE=1> <P> My postal address is:
-<P>Norman Megill
-<BR>93 Bridge St.
-<BR>Lexington, MA 02421-7946
-<BR> USA
+<ol>
+  <li>Via the <a href="mm-mailing-list.html" >metamath mailing list</a>.</li>
+  <li>By creating a github issue (most often in the
+  <a href="https://github.com/metamath/set.mm" >set.mm</a> repository but we
+  also list related repositories at <a href="other.html" >other.html</a> and
+  some are in the <a href="https://github.com/metamath" >the metmath github
```

Comment:
Nit: "metmath" should be "Metamath" here. Also, GitHub has capital G and H.

Reply:
Oooh, good catch on the missing "a".

As for capitalization, I wasn't sure how globally you meant this so I capitalized Metamath and GitHub everywhere in this file.  I don't have opinions myself (at least, not opinions strong enough to think about it very hard).